src/foundry/common/documents/tile.mjs.d.ts

```diff
@@ -0,0 +1,204 @@
+    data: object,
+    {
+      embedded
+    }?: {
+      /**
+       * Apply shims to embedded models?
+       * @defaultValue `true`
+       */
+      embedded?: boolean;
+    }
+  ): object;
+}
+export default BaseTile;
+
+declare namespace BaseTile {
+  type Metadata = Merge<
+    DocumentMetadata,
+    { name: "Tile"; collection: "tiles"; label: "DOCUMENT.Tile"; labelPlural: "DOCUMENT.Tiles" }
+  >;
+
+  type SchemaField = fields.SchemaField<Schema>;
+  type ConstructorData = UpdateData;
+  type UpdateData = fields.SchemaField.InnerAssignmentType<Schema>;
+  type Properties = fields.SchemaField.InnerInitializedType<Schema>;
+  type Source = fields.SchemaField.InnerPersistedType<Schema>;
+
+  interface Schema extends DataSchema {
+    /**
+     * The _id which uniquely identifies this Tile embedded document
+     * @defaultValue `null`
+     */
+    _id: fields.DocumentIdField;
+
+    /**
+     * An image or video texture which this tile displays.
+     * @defaultValue `null`
+     */
+    texture: TextureData<{}, { categories: ["IMAGE", "VIDEO"]; initial: null; wildcard: false }>;
+
+    /**
+     * The pixel width of the tile
+     * @defaultValue `0`
+     */
+    width: fields.NumberField<{
+      required: true;
+      min: 0;
+      nullable: false;
+      step: 0.1;
+    }>;
+
+    /**
+     * The pixel height of the tile
+     * @defaultValue `0`
+     */
+    height: fields.NumberField<{ required: true; min: 0; nullable: false; step: 0.1 }>;
+
+    /**
+     * The x-coordinate position of the top-left corner of the tile
+     * @defaultValue `0`
+     */
+    x: fields.NumberField<{ required: true; integer: true; nullable: false; initial: 0; label: "XCoord" }>;
+
+    /**
+     * The y-coordinate position of the top-left corner of the tile
+     * @defaultValue `0`
+     */
+    y: fields.NumberField<{ required: true; integer: true; nullable: false; initial: 0; label: "YCoord" }>;
+
+    /**
+     * The z-index ordering of this tile relative to its siblings
+     * @defaultValue `100`
+     */
+    z: fields.NumberField<{ required: true; integer: true; nullable: false; initial: 100 }>;
+
+    /**
+     * The angle of rotation for the tile between 0 and 360
+     * @defaultValue `0`
+     */
+    rotation: fields.AngleField;
+
+    /**
+     * The tile opacity
+     * @defaultValue `1`
+     */
+    alpha: fields.AlphaField;
+
+    /**
+     * Is the tile currently hidden?
+     * @defaultValue `false`
+     */
+    hidden: fields.BooleanField;
+
+    /**
+     * Is the tile currently locked?
+     * @defaultValue `false`
+     */
+    locked: fields.BooleanField;
+
+    /**
+     * Is the tile an overhead tile?
+     * @defaultValue `false`
+     */
+    overhead: fields.BooleanField;
+
+    /**
+     * Is the tile a roof?
+     * @defaultValue `false`
+     */
+    roof: fields.BooleanField;
+
+    /**
+     * The tile's occlusion settings
+     * @defaultValue `{ mode: CONST.TILE_OCCLUSION_MODES.FADE, alpha: 0, radius: null }`
```

Comment:
We *can* keep this as is, but in cases like these I found it easier to just write something that points the user to the inner properties. That way we don't have to update in two different places and end up forgetting one of them.

Reply:
Updated.